systemd/openqa-dump-db.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Executes openQA database dump and cleanup for openQA.
+After=postgresql.service openqa-setup-db.service
+Wants=openqa-setup-db.service
```

Comment:
Why would we need openqa-setup-db?

Reply:
Ok, understood. This should get executed as soon as `/var/lib/openqa` mount is available. `postgres` user is created during postgres db installation and not by our `openqa-setup-db.service`

Reply:
This is needed to ensure starting the timer unit on a fresh installation and every time at boot.

`openqa-setup-db.service` is dependent on `postgresql.service`, in our documentation, there's an instruction to start and enable `postgresql.service` https://open.qa/docs/#_run_the_web_ui

Reply:
I guess this was removed, hence resolving the thread